includes/functions.php

```diff
@@ -1177,6 +1177,11 @@ function generate_post_summary( $post, $length = 500 ) {
 		return '';
 	}
 
+	$content_warning = get_content_warning( $post->ID );
```

Comment:
if you set it in the transformer, you do not also have to add it here.

I quite like the idea to set it in the transformer only, this we can still generate the summary!

Reply:
If we would keep it in there, it would show the "content warning" also in the content field if someone would use the shortcode `[ap_excerpt]` for the content field.